src/scripts.js

```diff
@@ -77,6 +63,8 @@ const displayPantry = () => {
 const displayRandomFavorites = () => {
   let chunk = '';
   let fourRandomRecipes = [];
+
+  // TODO need to use one of the new prototype methods for this?
```

Comment:
I only want to run it 4 times, so it's a good use case for a for loop, as opposed to an iterator method.

Reply:
But I'm open to suggestions on iterator methods that could get only four random recipes.

Reply:
On our list to ask David about